mitre/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  connector-mitre:
+    image: opencti/connector-mitre:1.1.0
+    environment:
+      - RABBITMQ_HOSTNAME=rabbitmq
+      - RABBITMQ_PORT=5672 
+      - RABBITMQ_USERNAME=guest
+      - RABBITMQ_PASSWORD=guest
+      - MITRE_ENTERPRISE_FILE_URL=https://raw.githubusercontent.com/mitre/cti/master/enterprise-attack/enterprise-attack.json
+      - MITRE_ENTITIES=attack-pattern,course-of-action,intrusion-set,malware,tool
+      - MITRE_INTERVAL=7
+      - MITRE_LOG_LEVEL=info
```

Comment:
Missing confidence level?

Reply:
I think the confidence level for the MITRE will be discussed with them. MITRE provides hundreds of relationships between entities coming from many sources, we should think about this in more details to provide intelligent confidence_level generation depending on the source of the relationship, etc.